resources/metadatainfo.json

```diff
@@ -352,6 +366,13 @@
       "suffix": "objectTranslation",
       "xmlName": "CustomObjectTranslation"
     },
+    {
+      "directoryName": "objectTranslations",
```

Comment:
I'm not comfortable in editing this file manually. The content of the file should be the result of the `sfdx force:mdapi:describemetadata` command. 

Enabling entitlement on an org and running the describe command to update this file should fix the issue with EntitlementProcess.

For CustomObjectFieldTranslation this need to be handle in the code as this are some kind of child components that are no described and we have many others (LayoutTranslation, FieldSetTranslation ...)
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customobjecttranslation.htm

Reply:
@genoud Agreed! i will run sfdx force:mdapi:describemetadata to get the describe from org. is there a way we can respect the force ignore ?. we can find the list of metadata that aren't supported and ignore them instead of this promise failure. like this metadata isn't supported in the console.

due to the forceignore issue only i had to add a bypass flag in gitdiff. there are bunch of non salesforce files in the project. 
submodule
config files
.placeholders - inside empty folders
some scripts files
all these are added to forceignore to make all sfdx commands work.

Reply:
@genoud metdatainfo is updated with sfdx force:mdapi:describemetadata